public/javascripts/controllers/watchLectureVideos.js

```diff
@@ -284,17 +299,42 @@ function update_search_results() {
         $('#auto_scroll').change(function () {
             if (this.checked) {
                 autoScroll = true;
-
+                console.log("autoscroll on");
             } else {
                 autoScroll = false;
+                console.log("autoscroll off");
             }
             update_search_results();
         });
-
         player.on('playlistitem', function () {
             updateCurrentVideoTranscriptions();
         });
+        player.on('pause', function () {
+            data=create_json("pausevideo");
```

Comment:
need to capture more info, which video? at what timestamp?

Reply:
Refer to the sample table on the google doc to get an idea of all that is required